libcontainer/libcontainer_fuzzer.go

```diff
@@ -0,0 +1,48 @@
+// +build gofuzz
+
+package libcontainer
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"golang.org/x/sys/unix"
+)
+
+func FuzzInit(data []byte) int {
+	if len(data) < 5 {
+		return -1
+	}
+
+	pipe, err := os.OpenFile("pipe.txt", os.O_RDWR|os.O_CREATE, 0755)
+	if err != nil {
+		return -1
+	}
+	defer pipe.Close()
+	defer os.RemoveAll("pipe.txt")
+
+	consoleSocket, err = os.OpenFile("consoleSocket.txt", os.O_RDWR|os.O_CREATE, 0755)
+	if err != nil {
+		return -1
+	}
+	defer consoleSocket.Close()
+	defer os.RemoveAll("consoleSocket.txt")
+
+	var config *initConfig
```

Comment:
Why pointer?

Reply:
`linuxStandardInit` expects a pointer: https://github.com/opencontainers/runc/blob/master/libcontainer/standard_init_linux.go#L27

Reply:
`json.NewDecoder(reader).Decode(&config)` causes a pointer to a pointer, which doesn't seem to be expected

Reply:
That is the case here:
https://github.com/opencontainers/runc/blob/091dd32dd1fadb09141449295e4c012474fc19dc/libcontainer/init_linux.go#L80-L83